edivorce/apps/core/utils/efiling_documents.py

```diff
@@ -26,14 +26,14 @@ def forms_to_file(responses_dict, initial=False):
         if provide_marriage_certificate:
             uploaded.append({'doc_type': 'MC', 'party_code': 0})
 
-            if responses_dict.get('marriage_certificate_in_english') == 'NO':
-                uploaded.append({'doc_type': 'AFTL', 'party_code': 0})
-                uploaded.append({'doc_type': 'EFSS1', 'party_code': 1})
-
         elif not provide_certificate_later:
             uploaded.append({'doc_type': 'AFF', 'party_code': 0})
             uploaded.append({'doc_type': 'EFSS0', 'party_code': 0})
 
+        if responses_dict.get('marriage_certificate_in_english') == 'NO':
```

Comment:
I would add an extra part to the condition to be on the safe side and prevent possible misinterpretations, like so: 
`if provide_certificate_later and responses_dict.get('marriage_certificate_in_english') == 'NO':`

Looks good otherwise.